benchmark/README.rst

```diff
@@ -31,6 +31,18 @@ The script has three main functions.
   The performance plot shows the execution time of the benchmark as a function of a scalar
   "size parameter" ``n``, the exact meaning of which depends on each benchmark.
 
+Comparing Revisions
+-------------------
+
+An additional script ``benchmark_revisions.py`` enables the comparative benchmarking of different
+revisions of PennyLane. It calls upon ``benchmark.py`` and thus supports all its different arguments.
+The revisions -- including branches, tags and commits -- are specified via ``-r revision1[,revision2[,revision3...]]``,
+as in
+
+  python3 benchmark_revisions.py -r master,0c8e90a -d default.qubit,default.tensor time bm_mutable_rotations
```

Comment:
(At least right now I'd assume this renders like plain text)
```suggestion
  ``python3 benchmark_revisions.py -r master,0c8e90a -d default.qubit,default.tensor time bm_mutable_rotations``
```

Reply:
The code block is recommended :)